spec/features/collections/collection_results_spec.rb

```diff
@@ -55,13 +55,6 @@
     expect(page).to have_css('.badge-opendap')
   end
 
-  it "displays a placeholder for collections which have no thumbnail URLs" do
```

Comment:
This spec should still be here, but just assert the image is image_unavailable.svg